specification/batch/Azure.Batch/routes.tsp

```diff
@@ -0,0 +1,2237 @@
+import "@typespec/versioning";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.ClientGenerator.Core;
+
+namespace Azure.Batch;
+
+// Templates
+
+interface Ops {
+  #suppress "@azure-tools/typespec-azure-core/no-response-body" "Legacy behavior"
+  @access(Access.internal, "java")
```

Comment:
Decorator from `Azure.ClientGenerator.Core` (e.g. `@access`) should be defined in a `client.tsp`

e.g.
```ts
import "@azure-tools/typespec-client-generator-core";
import "./main.tsp";

using Azure.ClientGenerator.Core;

@@access(Azure.Batch.Ops.ListOperation, Access.internal, "java");
```

Reply:
added client.tsp